database/seeders/TrainCheckinSeeder.php

```diff
@@ -15,8 +15,7 @@ class TrainCheckinSeeder extends Seeder
      * @return void
      */
     public function run() {
-        for ($cnt = 0; $cnt < 50; $cnt++) {
-            $user = User::all()->random();
+        foreach (User::all() as $user) {
```

Comment:
I had to change the Seeder because the random generated data has thrown many Checkin Collisions for an random picked user. So I think it's better to check in only once per user within seeding. :)